scylla/src/frame/response/result.rs

```diff
@@ -499,6 +500,37 @@ fn deser_prepared_metadata(buf: &mut &[u8]) -> StdResult<PreparedMetadata, Parse
 
 fn deser_cql_value(typ: &ColumnType, buf: &mut &[u8]) -> StdResult<CqlValue, ParseError> {
     use ColumnType::*;
+
+    if buf.is_empty() {
+        match typ {
+            Boolean
+            | Counter
+            | Date
+            | Decimal
+            | Double
+            | Float
+            | Int
+            | BigInt
+            | Timestamp
+            | Inet
+            | List(_)
+            | Map(_, _)
+            | Set(_)
+            | UserDefinedType { .. }
+            | SmallInt
+            | TinyInt
+            | Time
+            | Timeuuid
+            | Tuple(_)
+            | Uuid
+            | Varint => return Ok(CqlValue::Empty),
```

Comment:
any reason against pattern-matching with a wildcard? `_ => Ok(CqlValue::Empty)`

Reply:
nothing, that was just to be explicit :) will change it